### DISCUSSION TITLE
Support for filter only and except in Group?

### DISCUSSION THREAD

**Discussion by YeeJiaWei** [tighten/ziggy#520] [General]
Any way or plan to support the group filtering with except and only?

<img width="669" alt="Screenshot 2022-02-28 at 10 50 27 AM" src="https://user-images.githubusercontent.com/21292986/155916073-7c173514-1c7b-439d-b64d-a6b09881d516.png">

**Comment by bakerkretzmar** [comment]
Love this idea 🙌🏻 would welcome a PR to add it, otherwise I'll take a stab at it at some point.

**Reply by YeeJiaWei** [reply]
Okay, i try to submit PR in this week
